components/core/tools/docker-images/clp-env-base-centos-stream-9/Dockerfile

```
@@ -1,46 +1,33 @@
 # syntax=docker/dockerfile:1
 
+# Empty default for the `ca_trust` build context, so the networked RUN steps below resolve their
+# mount when no host CA trust is supplied. `build.sh --with-ca-certs` overrides this stage with a
+# staged directory; without it the mount is empty, the guard falls through, and the image keeps its
+# own distro trust store. Nothing is ever baked in. See yscope-dev-utils exports/docker/ca-trust.
+FROM scratch AS ca_trust
+
 FROM dokken/centos-stream-9 AS base
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)